src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -375,7 +376,7 @@
 
   <Target Name="CopyRepoArtifacts"
           DependsOnTargets="RepoBuild"
-          Condition="'$(IsUtilityProject)' != 'true'"
+          Condition="'$(IsUtilityProject)' != 'true' and Exists('$(RepoManifestFile)')"
```

Comment:
Why do we need the Exists check here? We discussed in a previous PR that we don't want that as it could hide potential issues when a repo manifest accidentally didn't get created.

Reply:
This target is still needed until new `arcade` flows to installer. Until then, `source-build-reference-packages` and `arcade` repos are built with old arcade SDK, which produces repo manifest files. Without Exists check other repos would fail the `Error` task in this target, as they would not produce repo manifest files.

I'll add a comment for removal of `CoreRepoArtifacts` target and mention this.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18917/commits/9ccd9fef91977119e248048e95b7178e06691a86